views/content/dao-liquid-democracy.sol

```diff
@@ -52,7 +52,6 @@ contract LiquidDemocracy {
     function vote(address nominatedAddress) returns (uint voteIndex) {
         if (voterId[msg.sender]== 0) {
             voterId[msg.sender] = delegatedVotes.length;
-            numberOfVotes++;
```

Comment:
Why are you removing this?

Reply:
Because that line of code does nothing; the `numberOfVotes` variable is set to a different value two lines lower, overwriting this incremented value.

Reply:
@MidnightLightning 
How are you making sure that these changes don't cause any undesired effects?
Do you have some test suite for it yet?

I'm just asking because im just about to start creating some truffle 4.0.0-beta tests for these DAO examples.
So if you have something in place, it would be helpful if you could share it somehow.

Reply:
No automated testing suite; I'm just testing it by using it as the basis for my own DAO contract and running it in the Remix Javascript virtual chain and private development chains, and any errors I tweak in my own DAO I reported back here as a fix for the upstream template.